src/toolbar/sticky/stickytoolbarview.js

```diff
@@ -44,15 +108,6 @@ export default class StickyToolbarView extends ToolbarView {
 	init() {
 		super.init();
 
-		/**
-		 * A dummy element which visually fills the space as long as the
-		 * actual toolbar is sticky. It prevents flickering of the UI.
-		 *
-		 * @private
-		 * @property {HTMLElement} ui.stickyToobar.StickyToolbarView#_elementPlaceholder
-		 */
-		this._elementPlaceholder = document.createElement( 'div' );
-		this._elementPlaceholder.classList.add( 'ck-toolbar__placeholder' );
 		this.element.parentNode.insertBefore( this._elementPlaceholder, this.element );
 
 		// Update sticky state of the toolbar as the window is being scrolled.
```

Comment:
Yes, this could become a followup. Still the performance is not a priority ATM, so we could postpone it a little bit. Will you create a followup, @oskarwrobel?